iommu_data_structures.adoc

```diff
@@ -1078,7 +1078,7 @@ The process to translate an `IOVA` is as follows:
   disallowed" (cause = 256).
 . If `ddtp.iommu_mode == Bare` and any of the following conditions hold then
   stop and report "Transaction type disallowed" (cause = 260); else go to step
-  18 with translated address same as the `IOVA`.
+  19 with translated address same as the `IOVA`.
 ..  Transaction type is a Translated request (read, write/AMO, read-for-execute)
     or is a PCIe ATS Translation request.
 ..  Transaction type is a PCIe "Page Request" Message.
```

Comment:
There is "process to translate IOVA" to follow on Page Request. So this line shouldn't be there but in the PCIe ATS Page Request handling section

Reply:
Agreed. Updated.

Reply:
Need to remove this one also:
Transaction type is a PCIe "Page Request" Message.

Reply:
Yes. Updated.